website/blog/2020-08-04-2.1.0.md

```diff
@@ -4,7 +4,7 @@ authorURL: "https://github.com/sosukesuzuki"
 title: "Prettier 2.1: new --embedded-language-formatting option and new JavaScript/TypeScript features!"
 ---
 
-This release adds new `--embedded-language-formatting` option, support new JavaScript/TypeScript features and a lot of bugfixes and improvements!
+This release adds a new `--embedded-language-formatting` option, supports new JavaScript/TypeScript features, and includes many bug fixes and improvements!
```

Comment:
Don't you want update the original changelog? You might want run the `draft-blog-post.js` script again.

Reply:
Sorry, I mean other changes from changelog, not this one.